pyaedt/filtersolutions_core/lumped_topology.py

```diff
@@ -0,0 +1,538 @@
+        self._dll.setLumpedComplexElementTuneEnabled.restype = c_int
+        self._dll.getLumpedComplexElementTuneEnabled.argtype = POINTER(c_bool)
+        self._dll.getLumpedComplexElementTuneEnabled.restype = c_int
+
+        self._dll.getLumpedCircuitResponseSize.argtype = POINTER(c_int)
+        self._dll.getLumpedCircuitResponseSize.restype = c_int
+        self._dll.getLumpedCircuitResponse.argtypes = [c_char_p, c_int]
+        self._dll.getLumpedCircuitResponse.restype = c_int
+
+    @property
+    def generator_resistor(self) -> str:
+        """Generator resistor. The default is ``50``.
+
+        Returns
+        -------
+        str
+        """
+        generator_resistor_string = self._dll_interface.get_string(self._dll.getLumpedGeneratorResistor)
+        return generator_resistor_string
+
+    @generator_resistor.setter
+    def generator_resistor(self, generator_resistor_string):
+        self._dll_interface.set_string(self._dll.setLumpedGeneratorResistor, generator_resistor_string)
+
+    @property
+    def load_resistor(self) -> str:
+        """Load resistor. The default is ``50``.
+
+        Returns
+        -------
+        str
+        """
+        load_resistor_string = self._dll_interface.get_string(self._dll.getLumpedLoadResistor)
+        return load_resistor_string
+
+    @load_resistor.setter
+    def load_resistor(self, load_resistor_string):
+        self._dll_interface.set_string(self._dll.setLumpedLoadResistor, load_resistor_string)
+
+    @property
+    def current_source(self) -> bool:
+        """Flag indicating if the current source in synthesized circuit is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        current_source = c_bool()
+        status = self._dll.getLumpedCurrentSource(byref(current_source))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(current_source.value)
+
+    @current_source.setter
+    def current_source(self, current_source: bool):
+        status = self._dll.setLumpedCurrentSource(current_source)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def first_shunt(self) -> bool:
+        """Flag indicating if shunt element(s) are first in the synthesized circuit.
+        If ``False``, series element(s) are first.
+
+        Returns
+        -------
+        bool
+        """
+        first_shunt = c_bool()
+        status = self._dll.getLumpedFirstElementShunt(byref(first_shunt))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(first_shunt.value)
+
+    @first_shunt.setter
+    def first_shunt(self, first_shunt: bool):
+        status = self._dll.setLumpedFirstElementShunt(first_shunt)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def bridge_t(self) -> bool:
+        """Flag indicating if the bridgeT topology in synthesized circuit is enabled.
```

Comment:
```suggestion
        """Flag indicating if the bridgeT topology in the synthesized circuit is enabled.
```
Use an article within the brief description when it helps with comprehension.